e2e-tests/README.md

```diff
@@ -0,0 +1,189 @@
+workspaces. Firefox doesn't have a headless mode of operation, so we can't
+simply turn off the UI.
+
+__Use a different driver__
+
+[pytest-selenium] provides the ability to run tests against [many other][test envs] browser environments -- consider using a different driver executable or external provider.
+
+__xvfb__
+
+Xvfb provides a fairly easily work around on Linux.
+
+
+On Linux:
+
+    Install Xvfb and run the tests with it's xvfb-run binary. For
+    example, if you run tests like::
+
+        $ py.test ...
+
+
+    You can switch to this to run with Xvfb::
+
+        $ xvfb-run py.test ...
+
+
+    This creates a virtual X session for Firefox to run in, and sets
+    up all the fiddly environment variables to get this working
+    well. The tests will run as normal, and no windows will open, if
+    all is working right.
+
+
+Getting involved as a contributor
+---------------------------------
+
+We love working with contributors to improve the Selenium test coverage on
+Socorro but it does require a few skills.  You will need to be familiar
+with Python, Selenium, and have a working knowledge of GitHub.
+
+If you are comfortable with Python, it's worth having a look at the Selenium
+framework to understand the basic concepts of browser-based testing and the
+page objects pattern.
+
+If you need to brush up on programming but are eager to start contributing
+immediately, please consider helping out by doing manual testing.  You can
+help find bugs in Mozilla [Firefox][firefox] or find bugs in the Mozilla web
+sites tested by the [Firefox Test Engineering][firefoxtesteng] team.  We have many projects that would be
+thrilled to have your help!
+
+To brush up on Python skills before engaging with us, [Dive Into Python][dive]
+is an excellent resource.  MIT also has [lecture notes on Python][mit] available
+through their open courseware.  The programming concepts you will need to know
+include functions, working with classes, and the basics of object-oriented
+programming.
+
+Questions are always welcome
+----------------------------
+While we take great pains to keep our documentation updated, the best source of
+information is those of us who work on the project.  Don't be afraid to join us
+in irc.mozilla.org #fx-test to ask questions about our Selenium tests.  Mozilla
+also hosts the #breakpad chat room to answer your general questions about
+the Socorro project.
+
+Writing Tests
+-------------
+
+If you want to get involved and add more tests, then there are just a few things
+we'd like to ask you to do:
+
+1. Use the [template files][GitHub Templates] for all new tests and page objects
+2. Follow our simple [style guide][Style Guide]
+3. Fork this project with your own GitHub account
+4. Make sure all tests are passing, and submit a pull request with your changes
+5. Always feel free to reach out to us and ask questions. We'll do our best to help get you started and unstuck.
+
+License
+-------
```

Comment:
In fact, all the above "soft skill like" blocks of text no longer make sense when this project is part of socorro as a piece of code and not a project description.